test/regression/2016-04-14-sdiv-2.c_

```diff
@@ -0,0 +1,25 @@
+// XFAIL: *
```

Comment:
Actually, this file belongs to @ccadar's commit cc267845c8a4f954cdf9402ade03196be16148ea. I have no idea why it is shown here..